netbout-web/src/test/java/com/netbout/email/EmMessagesITCase.java

```diff
@@ -45,9 +53,25 @@
  * @author Matteo Barbieri (barbieri.matteo@gmail.com)
  * @version $Id$
  */
+@SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
 public final class EmMessagesITCase {
 
     /**
+     * Send it.
+     */
+    private static final String SEND_IT = "send it";
+
+    /**
+     * Don't sent it.
+     */
+    private static final String DON_T_SEND_IT = "don't send it";
+
+    /**
+     * Randomizer.
+     */
+    private static final Random RANDOM = new SecureRandom();
```

Comment:
@pecko I don't think it is a good idea to generate random aliases. Let's stick with constant aliases defined inside the test method.